docs/data/charts/radar/radar.md

```diff
@@ -53,6 +53,18 @@ The radar chart displays a grid behind the series that can be configured with:
 
 {{"demo": "DemoRadar.js" }}
 
+## Axis values
+
+You can add labels to metrics with the `<RadarAxis />`.
+This component requires a `metric` prop and can be configured with:
+
+- `angle` The angle used to display labels. By default it's the one associated to the given metric.
```

Comment:
I don't understand what `By default it's the one associated to the given metric.` means